### DISCUSSION TITLE
DISCUSSION: DataFusion Meetup in Seattle WA

### DISCUSSION THREAD

**Discussion by alamb** [apache/datafusion#10348] [Ideas]
After our first meetup, (see https://github.com/apache/datafusion/discussions/8152 / https://github.com/apache/datafusion/discussions/8522) I have heard there may be interest in other meetups in different geographies. 

I would personally love to support such meetups (will promote, try to attend, can help find sponsors, etc) as much as possible but I don't have the bandwidth to actually organize them.

Anyone who is interested in attending or helping to organize please leave a comment

Other meetup discussions:
https://github.com/apache/datafusion/discussions/10347

**Comment by phillipleblanc** [comment]
I'll be traveling to Seattle on October 14th (Mon) and 15th (Tue). How about planning a meetup on Monday Oct 14th?

**Reply by alamb** [reply]
FYI @westonpace  @sadboy and @viirya  who I think are all in this general area

**Reply by westonpace** [reply]
Indeed.  I'm in Olympia and can make a day trip to Seattle pretty much whenever.

**Reply by dennyglee** [reply]
Glad to support this meetup in October if that helps, eh?!

**Comment by lukekim** [comment]
We (Spice AI) plan to organize this meetup. We're going to aim for Mon, Oct 14, 6pm.

**Reply by dennyglee** [reply]
Oh that's so cool - let me know if I can help @lukekim  :)

**Comment by tshauck** [comment]
I'm in Shoreline (just north of Seattle), and will try to attend. Looking forward to it!

**Comment by eadgbear** [comment]
I'll be there for sure

**Comment by lukekim** [comment]
The event page is now up! https://lu.ma/tnwl866b

Call for speakers: https://forms.gle/WmJrqipu9JEXABug7

**Reply by phillipleblanc** [reply]
Reminder that this is happening on Monday! If you are in the Seattle area, come hang out with us!

**Comment by alamb** [comment]
This meetup was held and I think went great 

We are discussing the next one here:
- https://github.com/apache/datafusion/discussions/13500
